spec/features/consumer/shopping/cart_spec.rb

```diff
@@ -56,9 +56,14 @@
       let(:variant) { product_tax.variants.first }
 
       before do
+        Spree::Config.set allow_backorders: false
         add_product_to_cart order, product_tax
       end
 
+      after do
+        Spree::Config.set allow_backorders: true
```

Comment:
[RSpec around](https://relishapp.com/rspec/rspec-core/v/3-0/docs/hooks/around-hooks#invoke-the-example-using-`run()`) is meant for these cases. You would have:

```ruby
around(:example) do |example|
  Spree::Config.set allow_backorders: false
  example.run
  Spree::Config.set allow_backorders: true
end
```

But I wonder why a simple `allow(Spree::Config).to receive(:set).with(:allow_backorders).and_return(false)` doesn't work in the first place, as @enricostano suggested :thinking: